Xamarin.PropertyEditing.Mac/Controls/Custom/UnfocusableTextField.cs

```diff
@@ -59,14 +59,10 @@ public UnfocusableTextField (CGRect frameRect, string text) : base (frameRect)
 
 		private void SetDefaultTextProperties ()
 		{
-			this.label = new NSTextField {
+			this.label = new BasePropertyControlTextField {
 				AccessibilityElement = false,
 				BackgroundColor = NSColor.Clear,
 				Bordered = false,
-				Cell = {
-					LineBreakMode = NSLineBreakMode.TruncatingTail,
-					UsesSingleLineMode = true,
-				},
```

Comment:
Shouldn't this have meant the property editor labels would have had a ... and all that? They weren't and still don't, but should.

Reply:
They do, when clipped "correctly". Look at the `Type` field in the header Panel, when you resize to a narrow width. The `...` is there.

Reply:
So we need to fix the clipping separately? 👍

Reply:
Yes. Please see my comment on https://github.com/xamarin/Xamarin.PropertyEditing/pull/529